internal/common/hashtool.go

```diff
@@ -24,11 +28,13 @@ func HashCommit(values []*big.Int, issig bool) *big.Int {
 	for i, v := range values {
 		tmp[i+offset] = v.Go()
 	}
-	r, _ := asn1.Marshal(tmp)
+	r, err := asn1.Marshal(tmp)
+	if err != nil {
+		panic(err) // Marshal should never error, so panic if it does
```

Comment:
since this change the hashtool test is broken. at least on my machine. breaks on line 23. as soon as i comment out the panic, the test passes